app/views/smart_proxies/show.html.erb

```diff
@@ -49,12 +49,29 @@
         </div>
         <div class="row">
           <div class="col-md-4">
-            <%= _('Features') %>
+            <%= _('Active Features') %>
           </div>
           <div class="col-md-8">
             <%= @smart_proxy.features.to_sentence %> <%= refresh_proxy_icon(@smart_proxy, authorizer) %>
           </div>
         </div>
+        <% if @smart_proxy.has_feature?('Logs') %>
+          <div class="row">
+            <div class="col-md-4">
+              <%= _('Disabled Features') %>
```

Comment:
I think it would be better to call this line "Failed Features", since "Disabled" might imply that they are simply set to `enable: false` in the configuration rather then failed to load.